processing/src/main/java/org/apache/carbondata/processing/util/CarbonDataProcessorUtil.java

```diff
@@ -267,14 +267,24 @@ private static String getComplexTypeString(DataField[] dataFields) {
     return dimString.toString();
   }
 
+  private static String isDictionaryType(CarbonDimension dimension) {
```

Comment:
why this method return type is string??? Change this to string

Reply:
This is used to append in a String field.